tests/framework/extensions/clusters/clusters.go

```diff
@@ -68,6 +68,27 @@ func IsHostedProvisioningClusterReady(event watch.Event) (ready bool, err error)
 	return false, nil
 }
 
+// Verify if a serviceAccountTokenSecret exists or not in the cluster.
+func CheckServiceAccountTokenSecret(client *rancher.Client, clusterName string) (success bool, err error) {
+	clusterID, err := GetClusterIDByName(client, clusterName)
+	if err != nil {
+		return false, err
+	}
+
+	cluster, err := client.Management.Cluster.ByID(clusterID)
+	if err != nil {
+		return false, err
+	}
+
+	if cluster.ServiceAccountTokenSecret != "" {
+		fmt.Printf("serviceAccountTokenSecret in this cluster is: %s\n", cluster.ServiceAccountTokenSecret)
```

Comment:
Markus no print statements please. Use this package for logging `logrus.Infof()`